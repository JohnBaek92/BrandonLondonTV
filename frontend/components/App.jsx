import React from 'react';
import { Route } from 'react-router-dom';
import Header from './header';
import Footer from './footer';
import LandingPage from './landing';
import About from './about';

const App = () => (
  <div className="Site">
    <Header />
    <div className="Site-content">
      <Route exact path="/" component={LandingPage} />
      <Route exact path="/about" component={About} />
    </div>
    <Footer />
  </div>
);

export default App;
