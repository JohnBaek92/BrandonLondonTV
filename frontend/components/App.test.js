import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import App from "./App";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

test("App component should render as expected", () => {
  const component = shallow(<App />);
  const tree = toJson(component);

  expect(tree).toMatchSnapshot();
});
