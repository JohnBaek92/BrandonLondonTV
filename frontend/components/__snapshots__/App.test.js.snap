// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render as expected 1`] = `
<HashRouter>
  <div
    className="Site"
  >
    <Header />
    <div
      className="header-space"
    />
    <div
      className="container Site-content"
    >
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/about"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/joso"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/press"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/blog"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/blog/:id"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/london-athletic"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/contact"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/recent-work"
      />
    </div>
    <Footer />
  </div>
</HashRouter>
`;
